examples/terraform-submodules/gke/module.tf

```diff
@@ -16,6 +16,9 @@
 // Run:
 //  terraform apply -var project="<YOUR_GCP_ProjectID>" [-var agones_version="1.17.0"]
 
+// To enable agones-metrics add below command in `terraform apply`:
+// -var enable_agones_metrics_nodepool=true
+
```

Comment:
```suggestion
```

I think this can be rolled back - we don't have any other variable specific docs in a comment here.

Reply:
> I think this can be rolled back - we don't have any other variable specific docs in a comment here.

Thanks for the review and suggestions.
Done!